analyzers/tests/SonarAnalyzer.UnitTest/Extensions/KnownReferenceTests.cs

```diff
@@ -0,0 +1,295 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2023 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using Moq;
+using SonarAnalyzer.Common;
+using SonarAnalyzer.Extensions;
+using static SonarAnalyzer.Helpers.KnownReference;
+using static SonarAnalyzer.Helpers.KnownReference.Predicates;
+
+namespace SonarAnalyzer.UnitTest;
```

Comment:
Thsi file should live in `Helpers` directory, same as the class it tests.
And have `.Helpers` namespace
```suggestion
namespace SonarAnalyzer.UnitTest.Helpers;
```